substrate/primitives/blockchain/src/lib.rs

```diff
@@ -24,3 +24,5 @@ mod header_metadata;
 pub use backend::*;
 pub use error::*;
 pub use header_metadata::*;
+
+const LOG_TARGET: &str = "db";
```

Comment:
Shouldn't it be something like `blockchain` instead?

Reply:
Changed this to `db::blockchain`. We should keep the `db` prefix IMO since its used in code that is semantically related, even though in a different crate.